manifest.json

```diff
@@ -125,7 +126,6 @@
       "js/config.js",
       "js/core.js",
       "js/background/background.js"
-    ],
-    "persistent": false
```

Comment:
I think I have to remove this, in order to make it work properly, but I'm not so sure.